glean-core/rlb/src/private/ping.rs

```diff
@@ -2,15 +2,24 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use std::sync::{Arc, Mutex};
+
 use inherent::inherent;
 
+type BoxedCallback = Box<dyn FnOnce(Option<&str>) + Send + 'static>;
```

Comment:
Why does it need to be Send? I think I understand why everything else is there.

Reply:
Because [sometimes we send around pings](https://github.com/mozilla/glean/blob/before-next-submit/glean-core/rlb/src/lib.rs#L492-L494).
And because we store the closure within the ping we need to ensure it can be safely send (that is: it doesn't hold on to data that can't be send, such as local references).